doc/administrator/administrate.rst

```diff
@@ -197,22 +141,51 @@ To print the layers in high quality, you should add the following ``Metadata``:
 
    See also: :ref:`administrator_administrate_metadata`, :ref:`administrator_administrate_ogc_server`.
 
-
 LayerGroup
 ----------
 
 Attributes:
 
-* ``Name``: used throw OpenLayers.i18n to display the name on the layers tree.
-* ``Order``: used to order the layers and group on the layer tree.
-* ``Metadata URL``: optional (deprecated in v2).
-* ``Expanded``: is expanded on the layer tree by default (deprecated in v2).
-* ``Internal WMS``: if true it can include only ``Internal WMS`` layers,
-  if false it can include only ``external WMS`` or ``WMTS`` layers (deprecated in v2).
-* ``Group of base layers``: if not ``Internal WMS`` replace radio button by check box (deprecated in v2).
+* ``Name``: It is used throw the i18n tools to display the name on the layers tree.
+* ``Metadata URL``: (deprecated).
+* ``Description``: a description.
+* ``Expanded``: (deprecated).
+* ``Internal WMS``: (deprecated).
+* ``Base layers group``: (deprecated).
+* ``Children``: the ordered children elements.
+* ``Metadata``: Additional metadata.
+
+.. TODO: remove the deprecated fields if it's correctly migrated.
 
 Background layers
 -----------------
 
 The background layers are configured in the database, with the layer group named
 **background** (by default).
+
+Theme
+-----
+
+* ``Name``: It is used throw the i18n tools to display the name on the layers tree.
+* ``Metadata URL``: (deprecated).
+* ``Description``: a description.
+* ``Public``: makes the layer public, also use the ``Restriction areas`` to make it accessible.
+* ``Icon``: the icon URL.
+* ``Interfaces``: visible in following interface.
+* ``Restricted Roles``: Restricted to the following roles.
+* ``Functionalities``: The linked functionalities.
+* ``Children``: the ordered children elements.
+* ``Metadata``: Additional metadata.
+
+.. TODO: remove the deprecated fields if it's correctly migrated.
+
+Restricted area
+---------------
+
+Area where a specific role hase a permition to see (and evantualy to change) the objects.
```

Comment:
Suggestion: "A restricted area is an area for which a user must possess a specific role in order to be able to see (and edit, if applicable) the objects within this area."